lib/analytics.js

```diff
@@ -672,6 +689,7 @@ Analytics.prototype.normalize = function(msg) {
 
   // Ensure all outgoing requests include page data in their contexts.
   msg.context.page = defaults(msg.context.page || {}, pageDefaults());
+  msg.context.doNotTrack = Analytics._doNotTrack();
```

Comment:
I'm guessing it just needs to be added here? https://segment.com/docs/spec/common/#context-fields-automatically-collected